homeassistant/components/owntracks/__init__.py

```diff
@@ -180,7 +188,73 @@
             return json_response([])
 
     hass.helpers.dispatcher.async_dispatcher_send(DOMAIN, hass, context, message)
-    return json_response([])
+
+    response = []
+
+    if context.friends:
+        persons = (
+            hass.data[PERSON_DOMAIN].storage_persons
```

Comment:
You're not allowed to use `hass.data` from other integrations. You can only use data from the state machine.